stock_location_restrict_group/models/stock_location.py

```diff
@@ -26,7 +26,6 @@ def _compute_restricted_group(self):
         for location in self.filtered(lambda l: l.group_restricted):
             quants = self.env['stock.quant'].search(
                 [('location_id', '=', location.id)])
-            location.restricted_group = False
             rest_group = quants.mapped('history_ids').filtered(
```

Comment:
What I meant is that if you already have a restricted_group why to recompute it? As I see it, once you have a restricted_group computed for a location you do not want it to change because then it can lead to block procurement groups it was accepting before (quite unlikely in the other hand, I know, but just in case). In my opinion just <code>if not location.restricted_group:</code> after the for loop will do the work.